setup.py

```diff
@@ -1,57 +1,58 @@
-from setuptools import setup, find_packages
+#!/usr/bin/env python
 import os
-import cms
+
+from cms import __version__
+from setuptools import find_packages, setup
+
+
+REQUIREMENTS = [
+    'Django>=2.2,<4',
```

Comment:
Should this be less than 4 or less than and equal to 3.2 as Django have committed to 3.0, 3.1 and 3.2, 4.0, 4.1, 4.2.

Reply:
@yakky true but on the other hand restricting doesn't gain much either. That way people can try out django/master earlier without locally modifying the files.